Domain/Enums/UserRole.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Enums
+{
+    public enum UserRole
+    {
+        User,
```

Comment:
please specify values, at least for the first one =1, so second will have n+1 by default

Reply:
By default enums start with 0. Do you want me to make it start with 1?

Reply:
yes, if I do not setup anything it will be 0, if it is intended, so I am Ok. I looked again, so I am ok to satart with 0, because each user is in User Role by default